docs/user/merging-metadata.md

```diff
@@ -0,0 +1,93 @@
+# Merging metadata {#merging-metadata}
+
+## About merging metadata {#about-merging-metadata}
+
+## How to Merge Indicators {#indicators-merge}
+
+The indicator merge feature lets you combine multiple source indicators into a single target indicator. Follow these steps to merge indicators:
+
+1. **Go to the Indicators List:**
+   Click on **Indicators -> Indicator** in the left-hand side (LHS) menu.
+
+2. **Select Source Indicators:**
+   Check the boxes next to the indicators you want to merge.
+
+3. **Click the Merge Button:**
+   In the multi-select toolbar at the top, click the **Merge** button.
+
+4. **Review Selected Indicators:**
+   If needed, adjust the list of source indicators to include or remove any selections.
+
+5. **Choose a Target Indicator:**
+   Select the indicator you want to merge into.
+
+6. **Decide on Source Indicator Deletion:**
+   Choose whether to delete the source indicators after the merge.
+
+7. **Enter the Confirmation Code:**
+   Type the confirmation code displayed in the confirmation box.
+
+8. **Complete the Merge:**
```

Comment:
do we want to mention the need to enter the confirmation code (applies throughout)